website/docs/plugins/ray_launcher.md

```diff
@@ -35,35 +35,38 @@ An example using the ray local launcher by default is provided in the plugin rep
 $ pwd
 <project_root>/hydra_ray_launcher/example/
 
-$ python train.py hydra/launcher=ray_local random_seed=1 -m
- [HYDRA] Ray Launcher is launching 1 jobs, sweep output dir: multirun/2020-05-18/15-00-07
- [HYDRA] Ray not initialized, init with config: {'num_cpus': 1, 'num_gpus': 0}
-  INFO resource_spec.py:204 -- Starting Ray with 9.03 GiB memory available for workers and up to 4.54 GiB for objects. You can adjust these settings with ray.init(memory=<bytes>, object_store_memory=<bytes>).
- INFO services.py:1168 -- View the Ray dashboard at localhost:8265
- [HYDRA]        #0 : mode=local random_seed=1
-(pid=19442) [__main__][INFO] - Start training...
-(pid=19442) [model.my_model][INFO] - Init my model
+$ python train.py -m
+[2020-07-31 16:50:03,360][HYDRA] Ray Launcher is launching 1 jobs, sweep output dir: multirun/2020-07-31/16-50-02
+[2020-07-31 16:50:03,360][HYDRA] Initializing ray with config: {'num_cpus': 2, 'num_gpus': 0}
+2020-07-31 16:50:03,371 INFO resource_spec.py:204 -- Starting Ray with 8.64 GiB memory available for workers and up to 4.34 GiB for objects. You can adjust these settings with ray.init(memory=<bytes>, object_store_memory=<bytes>).
+2020-07-31 16:50:03,749 INFO services.py:1168 -- View the Ray dashboard at localhost:8265
+[2020-07-31 16:50:04,302][HYDRA]        #0 : random_seed=1
+(pid=45515) [2020-07-31 16:50:04,614][__main__][INFO] - Start training...
+(pid=45515) [2020-07-31 16:50:04,615][model.my_model][INFO] - Init my model
+(pid=45515) [2020-07-31 16:50:04,615][model.my_model][INFO] - Created dir for checkpoints. dir=/Users/jieru/workspace/hydra-fork/hydra/plugins/hydra_ray_launcher/example/multirun/2020-07-31/16-50-02/0/checkpoint
 ```
+
 Override `ray_init_cfg` to start ray with specific `ray.init()` config:
 ```commandline
-$ python train.py hydra/launcher=ray_local mode=local hydra.launcher.params.ray_init_cfg.num_cpus=2 random_seed=1 -m
+$ python train.py hydra.launcher.params.ray_init_cfg.num_cpus=2 random_seed=1 -m
```

Comment:
1. Use explicit --multirun, -m can be confusing to newcomers.
2. No need to demo overriding Ray parameters in the primary example.
3. To document the config, you can follow the example in [Submitit](https://hydra.cc/docs/next/plugins/submitit_launcher#usage), in particular 
```
python foo.py hydra/launcher=submitit_slurm --cfg hydra -p hydra.launcher
```

```suggestion
$ python train.py --multirun random_seed=1,2,3
```

Reply:
fixed. love the submitit launcher example.